Integrations/GoogleVault/GoogleVault.py

```diff
@@ -202,7 +202,8 @@ def create_hold_query(hold_name, corpus, accounts, terms, time_frame="", start_t
     if mail_query:
         request['query'] = {'mailQuery': mail_query}  # Adding the ready mail query
     for acc_id in accounts:
-        accounts_for_query.append({'accountId': acc_id})
+        acc_entry = {'accountId': acc_id} if '@' not in acc_id else {'email': acc_id}
```

Comment:
not too familiar with accounts in google, but are we sure they cannot contain `@`?
if so, the fix is fine, if not, it is better to detect if `acc_id` is actually an email address by using the `emailRegex` we have in common server

Reply:
Account IDs can only include numbers so I guess regex was needed here.